src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -439,10 +439,42 @@ public void initSendable(SendableBuilder builder) {
     SendableHelper.addChild(builder, this, armController, "ArmController");
     SendableHelper.addChild(builder, this, elevatorController, "ElevatorController");
     builder.addDoubleProperty(
-        "Elevator Position", elevatorLeftEncoder::getPosition, elevatorLeftEncoder::setPosition);
-    builder.addDoubleProperty("Arm Position", armEncoder::getPosition, armEncoder::setPosition);
-    builder.addBooleanProperty("Done Scoring", this::doneScoring, null);
+        "Elevator Position (in)",
+        elevatorLeftEncoder::getPosition,
+        elevatorLeftEncoder::setPosition);
+    builder.addDoubleProperty(
+        "Elevator Vel (in)", elevatorLeftEncoder::getVelocity, elevatorLeftEncoder::setPosition);
+    builder.addDoubleProperty(
+        "Elevator Left Abs Pos (deg)", elevatorLeftAbsEncoder::getPosition, null);
+    builder.addDoubleProperty(
+        "Elevator Right Abs Pos (deg)", elevatorRightAbsEncoder::getPosition, null);
+    builder.addBooleanProperty("Clear of intake", this::clearOfIntakeZone, null);
+    builder.addDoubleProperty(
+        "Arm Abs Position (deg)", armAbsoluteEncoder::getPosition, armEncoder::setPosition);
```

Comment:
I ~~know~~ edit: think we can't setPosition with the abs encoder, so I think it makes more sense to just not have a setter. It feels a little misleading.

Reply:
I think we can, it's just not really advisable.. but it's hard to accidentally set things anyway (didn't we figure out it only works in test mode?)

Reply:
Yup, that's true. In that case I think it's fine.